python/process_graphs/burntmapping_chunks.json

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
This is a not very useful process to have as an example here as it runs only on a single back-end and doesn't use any pre-defined processes from openEO.